roles/lib/files/FWO.Report.Filter/DynGraphqlQuery.cs

```diff
@@ -25,7 +25,10 @@ public class DynGraphqlQuery(string rawInput)
         public List<string> QueryParameters { get; set; } =
         [
             " $limit: Int ",
-            " $offset: Int "
+            " $offset: Int ",
+            " $active: Boolean ",
```

Comment:
Here the global parameters are set for all reports. This will collide with the parameters further down e.g TimeFilter and break the Changes Report